stl/inc/xmemory

```diff
@@ -73,12 +73,13 @@ _INLINE_VAR constexpr size_t _New_alignof = (_STD max)(alignof(_Ty),
 
 // STRUCT _Default_allocate_traits
 struct _Default_allocate_traits {
-    __declspec(allocator) static void* _Allocate(const size_t _Bytes) {
+    __declspec(allocator) static _CONSTEXPR20_DYNALLOC void* _Allocate(const size_t _Bytes) {
         return ::operator new(_Bytes);
     }
 
 #ifdef __cpp_aligned_new
-    __declspec(allocator) static void* _Allocate_aligned(const size_t _Bytes, const size_t _Align) {
+    __declspec(allocator) static _CONSTEXPR20_DYNALLOC
+        void* _Allocate_aligned(const size_t _Bytes, const size_t _Align) {

```

Comment:
I think I added it to the commit, but for some reason aligned allocation do not currently work with clang so this should fall back to unaligned `::operator new(_Ptr);` during constant evaluation